github-actions/scan/src/launcher.ts

```diff
@@ -84,17 +87,11 @@ export const LAUNCHER_CONTEXT_DEFAULTS: LaunchContext = {
  * Creates the initial launch context
  * @returns launch context
  */
-function createContext(): LaunchContext {
+async function createContext(): Promise<LaunchContext> {
```

Comment:
Why did you change to asynchronous? The method is only used internally to create the context - and the caller is already asynchronous.

Was there any special purpose?

Reply:
take a look at line 94
the "getClientVersion" method returns a Promise
We have to wait for the version before continuing the procedure therefore we need to "await"
In JavaScript any "await" keyword has to be surrounded by a "async" function